services/scabbard/libscabbard/src/service/v3/arguments.rs

```diff
@@ -17,15 +17,17 @@ use std::convert::TryFrom;
 use splinter::error::InvalidArgumentError;
 use splinter::service::ServiceId;
 
+use crate::store::service::ConsensusType;
```

Comment:
Commit message should say "to use ConsensusType"

Reply:
Whoops, fixed